Form-Controls/index.html

```diff
@@ -1,28 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>My form exercise</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+    <form>
+      <fieldset>
+        <label for="name">Name</label>
+        <input type="text" id="name" placeholder="Enter your name" required>
+        <label for="email">Email</label>
+        <input type="email" id="email" placeholder="Email" required>
+        <label for="color">Color</label>
+        <select name="color" id="color" required>
+          <option value="red">Red</option>
+          <option value="green">Green</option>
+          <option value="light-green">Light Green</option>
+        </select>
+        <label for="size">Sizes</label>
+        <select name="sizes" id="size" required>
+          <option value="XS">XS</option>
```

Comment:
You did great Job in this label. But I have a question, If I submit form by mistake without choosing my size, form would be submitted with the first Size (XS), How we can solve this problem?